generated/3.0.10/bookworm/Dockerfile

```diff
@@ -29,39 +29,44 @@ ENV FIREBIRD_MAJOR=3
 #   FB 3.0 uses libncurses5: https://github.com/FirebirdSQL/firebird/issues/6418#issuecomment-826245785
 RUN set -eux; \
     apt-get update; \
+    # Install prerequisites + curl for download
     apt-get install -y --no-install-recommends \
         libatomic1 \
         libicu72 \
         $([ $FIREBIRD_MAJOR -eq 3 ] && echo 'libncurses5' || echo 'libncurses6') \
         libtomcrypt1 \
         libtommath1 \
         netbase \
-        procps; \
-    rm -rf /var/lib/apt/lists/*
-
-# Fix libtommath for FB 3.0 -- https://github.com/FirebirdSQL/firebird/issues/5716#issuecomment-826239174
-RUN [ $FIREBIRD_MAJOR -eq 3 ] && ln -sf /usr/lib/x86_64-linux-gnu/libtommath.so.1 /usr/lib/x86_64-linux-gnu/libtommath.so.0 || true
-
-# Download
-ADD --checksum="sha256:$FIREBIRD_RELEASE_SHA256" \
-    --chown=root:root \
-    --chmod=777 \
-    $FIREBIRD_RELEASE_URL \
-    /tmp/firebird-bundle.tar.gz
-
-# Extract, install, clean
-RUN set -eux; \
+        procps \
+        ca-certificates \
+        curl; \
+    \
+    # Download
+    echo "Downloading Firebird from $FIREBIRD_RELEASE_URL"; \
+    curl -fSL "$FIREBIRD_RELEASE_URL" -o /tmp/firebird-bundle.tar.gz; \
+    echo "Verifying checksum: $FIREBIRD_RELEASE_SHA256 for /tmp/firebird-bundle.tar.gz"; \
+    echo "$FIREBIRD_RELEASE_SHA256 /tmp/firebird-bundle.tar.gz" | sha256sum -c -; \
+    \
+    # Extract, install, clean
     cd /tmp; \
     tar --extract --file=firebird-bundle.tar.gz --gunzip --verbose --strip-components=1; \
     ./install.sh -silent; \
-    rm *.tar.gz *.sh *.txt \
-    # Remove unnecessary files
+    # Remove downloaded archive and temporary files from /tmp
+    rm firebird-bundle.tar.gz install.sh *.txt; \
```

Comment:
Aren't extracted files missed without deletion?

Reply:
You are right. I just found a remaining `buildroot.tar.gz` in the image.